manifests/g/Google/Android/SDKPlatformTools/33.0.3/Google.Android.SDKPlatformTools.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.Android.SDKPlatformTools
+PackageVersion: 33.0.3
+InstallerType: zip
+Installers:
```

```suggestion
Commands:
- adb
- fastboot
- dmtracedump
- etc1tool
- hprof-conv
Installers:
```
When I created the Package locally (didn't see your PR then) it added this Commands section. 
Maybe its needed? (but I am not that familar with WinGet)

---

This field is ignored when `InstallerType` is a zip.

---

Alright, nice to know.